src/components/token-family/TokenFamilyHeader.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { useMemo, useRef } from 'react';
+import { Animated, Easing, StyleSheet, View } from 'react-native';
+import FastImage from 'react-native-fast-image';
+import CaretImageSource from '../../assets/family-dropdown-arrow.png';
+import { useLatestCallback } from '../../hooks';
+import { ButtonPressAnimation } from '../animations';
+import TokenFamilyHeaderIcon from './TokenFamilyHeaderIcon';
+import { Text } from '@rainbow-me/design-system';
+import { ImgixImage } from '@rainbow-me/images';
+import { ThemeContextProps } from '@rainbow-me/theme';
+
+export const TokenFamilyHeaderAnimationDuration = 200;
+export const TokenFamilyHeaderHeight = 50;
+
+const sx = StyleSheet.create({
```

Comment:
When to use `sx` and when to use `cx`? What's the logic here I've used and seen both 😄

Reply:
there were places in the code where `sx` was already used so i changed `cx` to `sx` everywhere recently, sorry for confusions :D